ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDYesNoOption.java

```diff
@@ -21,4 +23,20 @@ public boolean toBoolean() {
     public static CCDYesNoOption valueOf(boolean value) {
         return value ? YES : NO;
     }
+
```

Comment:
Suggestion : The CCD model dont need to know about our domain. We can keep it separate. I didnt like mapper as component for a enum translation also. But its upto you.